framework/tlvf/src/src/tlvftypes.cpp

```diff
@@ -0,0 +1,81 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include <climits>
+#include <linux/if_ether.h>
+#include <tlvf/tlvftypes.h>
+
+namespace tlvf {
+std::string int_to_hex_string(const unsigned int integer, const uint8_t number_of_digits)
+{
+    // 'number_of_digits' represent how much digits the number should have, so the function will
```

Comment:
These comments should be moved to header file, inside a properly doxygen-formatted documentation

Reply:
Done.